incubator/cassandra/templates/backup/cronjob.yaml

```diff
@@ -0,0 +1,74 @@
+{{- if .Values.backup.enabled }}
+{{- $release := .Release }}
+{{- $values := .Values }}
+{{- $backup := $values.backup }}
+{{- range $index, $schedule := $backup.schedule }}
+---
```

Comment:
Not required.

Reply:
Hey @viglesiasce,

I`ve added all required changes.
Can you clarify which part here is not required?
If there is more than 1 keyspace to backup, multiple CronJobs will be created.

Thanks for the review!